src/Toggles.ts

```diff
@@ -47,6 +47,9 @@ export const toggleTabs = (name: keyof typeof tabNumberConst) => {
     revealStuff();
     hideStuff();
 
+    const el = document.activeElement as HTMLElement;
```

Comment:
this is nullable

Reply:
ok
"eslint-disable-next-line @typescript-eslint/no-unnecessary-condition" is also gone.